COLLABORATOR_GUIDE.md

```diff
@@ -122,12 +122,14 @@ Before landing pull requests, sufficient time should be left for input
 from other Collaborators. In general, leave at least 48 hours during the
 week and 72 hours over weekends to account for international time
 differences and work schedules. However, certain types of pull requests
-can be fast-tracked and may be landed after a shorter delay:
-
-* Focused changes that affect only documentation and/or the test suite.
-  `code-and-learn` and `good-first-issue` pull requests typically fall
-  into this category.
-* Changes that fix regressions.
+can be fast-tracked and may be landed after a shorter delay. For example:
+
+* Focused changes that affect only documentation and/or the test suite:
+  * `code-and-learn` tasks typically fall into this category.
+  * `good-first-issue` pull requests may also be suitable.
```

Comment:
How about:

>Pull requests from new contributors (for example `code-and-learn` tasks) that are low-risk and uncontentious.

- If the aim is to improve the experience of your first node contribution, then let's say that specifically.
- If we say docs or tests only, does that cover source comments? Or whitespace fixes? I feel like specifying the "what" is going to be too complex, but the "why" might be easier.